### PR TITLE
feat(ui-components) Add button component

### PR DESCRIPTION
**Pull request by robertpin**
Use the new button component in Participants Pane

### CODE REVIEW

**Comment by saghul** [comment]
I think this implementation should share an interface with https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/react/components/native/Button.tsx

File: react/features/base/components/common/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+const Button = ({
+    accessibilityLabel,
+    disabled,
+    fullWidth,
+    icon,
+    id,
+    onClick,
+    size = 'medium',
+    text,
+    type = 'primary'
```
Comment (Calinteodor) [inline react/features/base/components/common/Button.tsx:176]: Please use type object that we created in constants.js.

File: react/features/base/components/common/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+const Button = ({
+    accessibilityLabel,
+    disabled,
+    fullWidth,
+    icon,
+    id,
+    onClick,
+    size = 'medium',
+    text,
```
Comment (Calinteodor) [inline react/features/base/components/common/Button.tsx:175]: Can't we rely on types and not use this prop?

File: react/features/base/components/common/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        fullWidth: {
+            width: '100%'
+        }
+    };
+});
+
+const Button = ({
+    accessibilityLabel,
+    disabled,
+    fullWidth,
```
Comment (Calinteodor) [inline react/features/base/components/common/Button.tsx:170]: I think we can make the button have a full width by default and not rely on this prop.

**Comment by Calinteodor** [comment]
Also, can we update the context menu and ask to unmute buttons?

File: react/features/base/components/common/Button.tsx (hunk 3 above)
Comment (robertpin) [reply react/features/base/components/common/Button.tsx:170]: On web most buttons will not be full width, so I think it's better to have it like this

Comment (Calinteodor) [reply react/features/base/components/common/Button.tsx:166]: This means we can make it a prop with a false default value.

Comment (robertpin) [reply react/features/base/components/common/Button.tsx:166]: It is a prop with a falsy default value

Comment (Calinteodor) [reply react/features/base/components/common/Button.tsx:166]: You def right.

**Comment by robertpin** [comment]
> Also, can we update the context menu and ask to unmute buttons?

No. It's out of scope for this PR. The purpose of this PR is to add the new button component. Replacing all the buttons will follow in subsequent PRs

**Review by Calinteodor — APPROVED** [review]

**Pull request merged**